content/en/docs/concepts/traffic-management/index.md

```diff
@@ -307,7 +307,7 @@ spec:
 
   http:
   - match:
-      sourceLabels:
+    - sourceLabels:
         app: reviews
     route:
 ...
```

Comment:
Maybe it would be better to just remove this sourceLabels part, since it seems to have nothing to do with the example being described.